libbeat/metric/system/process/process.go

```diff
@@ -227,6 +227,20 @@ func getProcState(b byte) string {
 	return "unknown"
 }
 
+// GetOwnResourceUsageTime return the user and system CPU usage time in milliseconds
+func GetOwnResourceUsageTime() (int64, int64, error) {
+	r := sigar.Rusage{}
```

Comment:
nice :)

btw. the time is based on ticks.

Reply:
On linux getrusage is a syscall: https://github.com/torvalds/linux/blob/master/kernel/sys.c#L1660
 (also see: [task_cputime](https://github.com/torvalds/linux/blob/6363b3f3ac5be096d08c8c504128befa0c033529/kernel/sched/cputime.c#L865 and [vtime.h](https://github.com/torvalds/linux/blob/ead751507de86d90fa250431e9990a8b881f713c/include/linux/vtime.h#L52:13)).

Reply:
If this is returning "time" then I think it should use `time.Duration`.

Reply:
@urso I decided to include both because I read that a CPU tick can be a different time on Linux and Windows.